client/main.js

```diff
@@ -0,0 +1,22 @@
+
+const getSentenceButton = document.getElementById("get-sentence-btn")
+const sentenceContainer = document.getElementById("sentence-container")
+
+const serverAddress = "localhost"
+const serverPort = 3000
+const getSentenceRoute = "greateSentence"
+
+const getSentenceURI = `http://${serverAddress}:${serverPort}/${getSentenceRoute}`
+
+function getSentenceFromServer() {
+    window.fetch(new Request(getSentenceURI))
```

Comment:
no need for `window.`.
Anything defined on the window is accessible directly - `fetch(...)`.

Btw, the window has the whole global scope, including your functions and variables, and all other builtins (like Math for instance).
In nodejs, the `window` is called `global`